io.github.voxelcubes.panelcleaner.yaml

```diff
@@ -0,0 +1,32 @@
+id: io.github.voxelcubes.panelcleaner
+runtime: org.kde.Platform
+runtime-version: '6.6'
+sdk: org.kde.Sdk
+command: pcleaner-gui
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=home
+  - --share=network
+modules:
+  - pypi-dependencies.yaml
+  - manual-dependencies.yaml
+
+  - name: pcleaner
+    buildsystem: simple
+    build-commands:
+      - pip3 install --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST} pcleaner --no-build-isolation
+      - desktop-file-edit --set-icon=${FLATPAK_ID} PanelCleaner.desktop
+    sources:
+      - type: file
+        url: https://files.pythonhosted.org/packages/0e/b8/51277f2adceb29f6a606c3dcbe6ab5cf55a7a8fc93e53d1246a510faff31/pcleaner-2.1.4-py3-none-any.whl
+        sha256: ab1951a153b7dd6f22567646a08af3dc0bc1509ce69027d54d1153871c41778c
+      - type: git
+        url: https://github.com/VoxelCubes/PanelCleaner
+        commit: 03499c608438cb7a67e7655536488cf120bf3d35
```

Comment:
Is this just for the icon, desktop etc.? You can directly link to them from a tag/commit using type: file

Reply:
I've now changed it to pull the three files needed instead.